src/device/usbd.c

```diff
@@ -53,6 +54,8 @@ typedef struct {
     uint8_t self_powered          : 1; // configuration descriptor's attribute
   };
 
+  uint8_t speed;
```

Comment:
Maybe using `tusb_speed_t` type would fit a little better here?

Reply:
I am bit afraid the enum will cause the size of variable to 4 unnecessary

Reply:
Oh yeah, I always forget short enum are not gcc standard. Better keep it as uint8_t then.